### PR TITLE
HV:Acrn-hypvervisor Root Directory Clean-up and create misc/ folder f…

### PR DESCRIPTION
**Pull request by terryzouhao**
HV:Acrn-hypvervisor Root Directory Clean-up and create misc/ folder for Acrn daemons, services and tools.

This patch is to clean-up acrn-hypervisor root directory, targt only 5 folders under acrn-hypervisor:1.hypervisor,2.devicemodel,3.misc,4.doc,5.build

Tracked-On: #3482
Signed-off-by: Terry Zou <terry.zou@intel.com>
Acked-by: Eddie Dong <eddie.dong@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

**Comment by acrnsi** [comment]
OK to verify

File: misc/acrn-config/host/kconfig/test.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print('aaaaaaaaaaaa')
```
Comment (dongyaozu) [inline misc/acrn-config/host/kconfig/test.py]: why aaaa?

File: misc/acrn-config/host/kconfig/pylint.conf [hunk 2]
```diff
@@ -0,0 +1,570 @@
+[MASTER]
```
Comment (dongyaozu) [inline misc/acrn-config/host/kconfig/pylint.conf]: A new file?

File: misc/acrn-config/host/kconfig/test.py (hunk 1 above)
Comment (terryzouhao) [reply misc/acrn-config/host/kconfig/test.py]: Delete this print line, which was for temporary debug. 
PS: Actually kconfig/ folder was just copied from original acrn-hypervisor/scripts. pylint is only 7.76 score, i will try best to clean most basic coding guideline issues in this PR, and continue optimizing target to 10 score in another patch/PR.

File: misc/acrn-config/host/kconfig/pylint.conf (hunk 2 above)
Comment (terryzouhao) [reply misc/acrn-config/host/kconfig/pylint.conf]: It was pylint temporary file during scan, will delete it in source code.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: misc/acrn-config/host/kconfig/test.py (hunk 1 above)
Comment (dongyaozu) [reply misc/acrn-config/host/kconfig/test.py]: Yes, let us close this PR first. Revise to fix the issue and re-submit

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by terryzouhao** [comment]
Hi, i had two force-pushs from 91fd306 to 7398156 and d152d28. Updated/optimized below key issues:
a. removed temporary files in kconfig/:  test.py, pylint.conf.
b.optimized pylint score of kconfig/ Python scripts from 7.76 to 8.6.
c. updated sub-folder tree in acrn-config/ and fixed user-guides scripts in doc/ folder.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**